README.md

```diff
@@ -1,6 +1,16 @@
 # starknet-rivet
 
-> a chrome extension tools built with Vite + React, and Manifest v3
+Starknet Rivet is a developer wallet tailored specifically for the Starknet ecosystem. 
+
+It aims to streamline your development process, offering:
+
+1. Easy Onboarding: Set up a local devnet effortlessly
+2. Block and Transaction Explorer: Inspect blocks and transactions
+3. Account Management: View and manage Starknet accounts
+4. Contract Deployment: Deploy directly from the extension
+5. Devnet Configuration: Customize your development environment
+6. DApp Integration: Seamlessly connect to all Starknet dApps
+7. Enhanced Debugging: Efficient local testing and troubleshooting
```

Comment:
What do you mean by that ?
Users can use our rivet wallet to test their Dapp on devnet (is that what you mean ?)